<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>playlist</class>
 <widget class="QWidget" name="playlist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>484</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1076</width>
    <height>484</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	/*background-color: rgba(255, 255, 255, 0.6);*/
	background-color: transparent;
	color: #fff8ea;
	border: 2px solid white;
	font-size: 16px;
	font-weight: 700;
	font-family: ZCOOL KuaiLe;
}
/*歌单图片*/
#list_img_btn{
	border-radius: 15px;
}

/*上一页、下一页按钮*/
QPushButton[page]{
	border: 2px solid #ca8f00;
	border-radius: 15px;
}
QPushButton[page]:hover{
	font-size: 17px;
	background-color: #ca8f00;
}

/*垂直滚动条样式*/
QScrollBar::add-line, QScrollBar::sub-line {
    width: 0;  /* 隐藏水平滚动条的箭头 */
    height: 0; /* 隐藏垂直滚动条的箭头 */
}
QScrollBar::add-page, QScrollBar::sub-page {
    background: transparent;
    /*background: rgba(255, 255, 255, 0.5);*/
	border: 0px;
	border-radius: 2px;
	padding-top: 2px;
    padding-bottom: 2px;
}
QScrollBar{
    width: 10px;
    border: 0px;
    border-radius: 5px;
	padding: 2px;
}
QScrollBar::handle{
	width: 4px;
	border-radius: 2px;
	/*background-color: #84ACD9;*/
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #639AD4, stop: 1 #DE76A3);
}
QScrollBar::handle:hover{
	/*background-color: #75ACEB;*/
	background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 #639AD4, stop: 1 #DE76A3);
}
QScrollBar:vertical {
    border: none;          /* 去除滚动条整体边框 */
    background: transparent;  /* 背景透明（可选） */
}
QScrollArea .QWidget{
	border: none;
}
QScrollArea{
	padding-top: 5px;
	padding-bottom: 5px;
}
/*垂直滚动条样式*/

/*推荐页面通用样式*/
[recommend]{
	font-weight: 700;
	font-size: 30px;
}
[day]{
	font-size: 18px;
}
/*推荐页面通用样式*/


#play_list_btn{
	border-image: url(:/play/tools/resource/bg/icon/play/play.png);
}
#collect_playlist_btn{
	border-image: url(:/play/tools/resource/bg/icon/play/folder_add.png);
}
QPushButton[playlist]:hover{
	margin: 2px;
	border-radius: 5px;
	background-color: rgba(255, 255, 255, 0.4);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <property name="spacing">
      <number>25</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>15</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="list_img_btn">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="list_name_lab">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>未加载歌单（x首）</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="recommend" stdset="0">
          <string>-1</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="playlist_scroll_area">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1032</width>
        <height>343</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_24">
       <item>
        <layout class="QVBoxLayout" name="playlist_verticalLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>289</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
